src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -85,16 +86,18 @@
     /**
      * A buffer value to be sure to be outside of the colony.
      */
-    private static final int BUFFER = 10;
+    private static final int BUFFER                    = 10;
 
     /**
      * The last colony id.
      */
     private static int topColonyId = 0;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "topColonyId" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1068)